backend/src/utils/lambda_utils.py

```
@@ -105,4 +109,32 @@ def extract_user_id_from_event(event: Dict[str, Any]) -> Optional[str]:
+        
+    Returns:
+        Tuple of (validated_model, error_dict) where one will be None
+    """
+    body_str = event.get('body')
+    if not body_str:
+        return None, {"message": "Request body is missing or empty"}
+    
+    try:
+        return model_class.model_validate_json(body_str), None
+    except ValidationError as e:
+        logger.error(f"Validation error: {str(e)}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)